Control coverage/Feature/DBForPostgreSQL.md

```diff
@@ -0,0 +1,6 @@
+## DBForPostgreSQL
+
+| ControlId | Dependent Azure API(s) and Properties | Control spec-let |
+|-----------|-------------------------------------|------------------|
+| <b>ControlId:</b><br>Azure_DBforPostgreSQL_Audit_Enable_ATP<br><b>DisplayName:</b><br>Enable Threat detection for PostgreSQL<br><b>Description: </b><br> Advanced Threat Protection must be enabled for Azure Database for PostgreSQL |<b> ARM API to get security alert policy of a DBForPostgreSQL server: </b> <br> /subscriptions/{subscriptionId}<br>/resourceGroups/{resourceGroupName}<br>/providers/Microsoft.DBforPostgreSQL<br>/servers/{serverName}<br>/securityAlertPolicies/Default<br>?api-version=2017-12-01 <br><b>Properties:</b><br> properties.state| <b>Passed: </b><br>ATP is enabled and 'email notifications to admins' are also enabled.<br><b>Failed: </b><br>Either Postgre Sql is of 'basic tier' which does not support ATP. <br>Or ATP is disabled. <br>Or ATP is enabled but 'email notifications to admins' are disabled. |
```

Comment:
You may want to specify the other properties that are being considered for the evaluation - `emailAccountAdmins`.
Please replace all occurrences of `Postgre sql` with `PostgreSQL`.